plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -221,9 +222,9 @@ export const Stepper = (stepperProps: StepperProps) => {
   const mergedUiSchema = merge({}, propUiSchema, currentStep?.uiSchema);
 
   const handleCreate = useCallback(() => {
-    props.onCreate(stepsState);
+    onCreate(stepsState);
```

Comment:
if onCreate already returns a promise, isn't it better if the stepper just handles the disabled state based on that starting/completing? Should not need to pass that in as a prop

Reply:
`isCreating` is already there at `TemplateWizardPage` before this PR, so I chose not creating a new `isCreating` state at `Stepper` level.

Reply:
But the `Workflow` and `Stepper` components can be used outside of the `TemplateWizardPage`. That's just local state for that component I think that that point. It's to render the `Progress` which isn't available on other pages.

I'm kind of torn, but would opt to not add additional props to public components if we can avoid it in this case.

Reply:
I'm fine with both personally, but you two vote for local state, then I'll change as you advised. 😁